### DISCUSSION TITLE
Curation report 2023 Jan 30th - Feb 3rd

### DISCUSSION THREAD

**Discussion by fehrhart** [wikipathways/GPML-Repo#111] [Curation reports]
Hi all,

 not much to discuss, a looooot of changes to approve from Lars on references.  

@khanspers - is this one finished? https://www.wikipathways.org/index.php/Pathway:WP5293 Looks pretty complete except description missing :)

Duplicate - one of them is still in private mode but I guess one them will be deleted soon. The creator is a student of UM who will do an internship with me starting next week, so will catch her next week anyway.
https://www.wikipathways.org/index.php/Pathway:WP5319
https://www.wikipathways.org/index.php/Pathway:WP5318

Next one up is @mkutmon

**Comment by khanspers** [comment]
Thanks Freddie. WP5293 is still under construction, I didn't get a chance to complete it yet. 

And I'm resetting the admin rotation today, so look out for calendar invites.
